app/src/main/java/com/canvearth/canvearth/mapListeners/OnCameraIdleListenerImpl.java

```diff
@@ -26,14 +27,17 @@ public OnCameraIdleListenerImpl(MapsActivity activity, Context context, MapScale
     @Override
     public void onCameraIdle() {
         CameraPosition cameraPosition = MapsActivity.Map.getCameraPosition();
-        ScreenUtils.showToast(context, "lat: " + cameraPosition.target.latitude + "\n" +
-                "lng: " + cameraPosition.target.longitude + "\n" +
-                "zoom: " + cameraPosition.zoom);
+//        ScreenUtils.showToast(context, "lat: " + cameraPosition.target.latitude + "\n" +
```

Comment:
Is there any necessity of this toast message? If we don't need this anymore, why don't you just remove it?

Reply:
Got it